DeckPricer/Bot/CMakeLists.txt

```
@@ -52,4 +52,5 @@ target_include_directories(Bot
     $<INSTALL_INTERFACE:include>
 )
 
-target_link_libraries(Bot dpp)
\ No newline at end of file
+target_link_libraries(Bot dpp)
+target_link_libraries(Bot base64)

```

We can combine this into a single call :D
```suggestion
target_link_libraries(Bot dpp base64)
```